src/ripple/ledger/impl/View.cpp

```diff
@@ -391,7 +391,13 @@ transferRate (ReadView const& view,
     auto const sle = view.read(keylet::account(issuer));
 
     if (sle && sle->isFieldPresent (sfTransferRate))
-        return Rate{ sle->getFieldU32 (sfTransferRate) };
+    {
+        auto rate = (*sle)[sfTransferRate];
+        auto maxRate = 2 * QUALITY_ONE;
```

Comment:
I am convinced re: readability. The signedness is an issue that should be addressed in another commit in a better way.